TweetMate/src/main/java/com/ybsystem/tweetmate/fragments/dialog/PreviewAppearanceDialog.java

```diff
@@ -36,17 +33,10 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         // Inflate
         View view = inflater.inflate(R.layout.dialog_preview_appearance, null);
 
-        // Declare
-        final long TWITTER_JP_USER_ID = 7080152L;
-        boolean isPrivate = false;
-
-        // Create timeline
-        Fragment fragment = new ProfileTimeline().newInstance(
-                new Column(TWITTER_JP_USER_ID, "", USER_TWEET, false),
-                isPrivate
-        );
-
         // Set timeline
+        Fragment fragment =  new SearchTweetTimeline().newInstance(
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ybs32/TweetMate-for-Android/blob/64d6caf4400bbdfa265133edeb71ed500c13e3d4/TweetMate/src/main/java/com/ybsystem/tweetmate/fragments/dialog/PreviewAppearanceDialog.java#L37"></span>Avoid passing <code>null</code> as the view root (needed to resolve layout parameters on the inflated layout's root element)</td>
    </tr>
  </tbody>
</table>